migration/v2/model/dependency.go

```diff
@@ -11,9 +11,9 @@ var depMutex sync.Mutex
 
 type Dependency struct {
 	Model
-	ToID   uint         `gorm:"index"`
+	ToID   uint         `gorm:"uniqueIndex:dependency_primary_key"`
 	To     *Application `gorm:"foreignKey:ToID;constraint:OnDelete:CASCADE"`
-	FromID uint         `gorm:"index"`
+	FromID uint         `gorm:"uniqueIndex:dependency_primary_key"`
```

Comment:
Appreciate your effort on looking thi issue, but it cannot be in this PR with tests. Please create a new one referencing to reported issue and look on notes below.
- the schema is versioned, it is needed create new version (or modify the latest one if it was not released yet - I'm not sure if it is possible now, let's see on the new PR)
- I'm not sure about using "primary_key" as part of name of composite uniq key, would you consider a more descriptive name?

Reply:
Surely ! Will address it in the next PR